README.md

```diff
@@ -9,148 +9,38 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/shanilhirani/go-credly)](https://goreportcard.com/report/github.com/shanilhirani/go-credly)
 [![Codecov](https://codecov.io/gh/shanilhirani/go-credly/branch/main/graph/badge.svg)](https://codecov.io/gh/shanilhirani/go-credly)
 
-⭐ `Star` this repository if you find it valuable and worth maintaining.
+## About
 
-👁 `Watch` this repository to get notified about new releases, issues, etc.
+This repository contains a Go application that fetches public data from Credly's API which results in a users badges on Credly.
```

Comment:
I think this explanation could be a bit clearer, specifically, what's the output? Is it the badges public endpoints?